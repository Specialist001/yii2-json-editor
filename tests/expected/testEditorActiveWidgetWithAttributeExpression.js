modelmock1Data1JsonEditor_46acca2d = new JSONEditor(document.getElementById('modelmock-1-data-1-json-editor'), {"mode":"tree","onChange":function() {jQuery('#modelmock-1-data-1').val(modelmock1Data1JsonEditor_46acca2d.getText());}}, {"foo":"bar"});
jQuery('#modelmock-1-data-1').parents('form').submit(function() {jQuery('#modelmock-1-data-1').val(modelmock1Data1JsonEditor_46acca2d.getText());});