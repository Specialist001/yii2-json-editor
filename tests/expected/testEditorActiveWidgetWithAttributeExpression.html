<form id="data-form" action="test" method="post"><div class="form-group field-modelmock-1-data-1">
<label class="control-label" for="modelmock-1-data-1">Data</label>
<input type="hidden" id="modelmock-1-data-1" name="ModelMock[1][data][1]" value="{&quot;foo&quot;: &quot;bar&quot;}" data-json-editor-name="modelmock1Data1JsonEditor_46acca2d"><div id="modelmock-1-data-1-json-editor"></div>

<div class="help-block"></div>
</div></form>